lib/analyse.ml

```diff
@@ -191,7 +195,14 @@ module Analysis = struct
             Current.Process.check_output ~cwd:dir ~cancellable:true ~job cmd >>= begin function
               | Error _ ->
                 (* new release *)
-                Lwt_result.return (add_pkg ~path ~name ~package New pkgs)
+                let package_dir = "packages/"^name in
+                let cmd = "", [| "git"; "show"; master^":"^package_dir |] in
+                Current.Process.check_output ~cwd:dir ~cancellable:true ~job cmd >>= begin function
```

Comment:
Looks like we could get by with `exec` here, since we don't care about the output:

```suggestion
                Current.Process.exec ~cwd:dir ~cancellable:true ~job cmd >>= begin function
```

Not very important, but seems so save a fer characters, is a smaller change, and maybe saves an allocation. Feel free to ignore if you don't agree.

Reply:
Yes, I used `check_output` since we were using it for the check to detect new releases, but this makes sense. Thanks!